app/views/document_lead_image/edit.html.erb

```diff
@@ -8,7 +8,12 @@
   no_border: true
 } %>
 
-<%= form_tag(update_document_lead_image_path(@document, @image, next_screen: params[:next_screen]), method: :patch) do %>
+<% submit_path = case params[:wizard]
+                 when "lead_image" then update_and_choose_document_lead_image_path(@document, @image)
+                 else update_document_lead_image_path(@document, @image)
+                 end %>
```

Comment:
Feels like this case should just be an if?

Reply:
Sure - updated to a ternary :-) I was thinking we may have other wizards in future...